DigitalLearningSolutions.Web.Tests/Controllers/ForgotPassword/ForgotPasswordControllerTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.ForgotPassword
+{
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers;
+    using DigitalLearningSolutions.Web.ViewModels.ForgotPassword;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using NUnit.Framework;
+
+    class ForgotPasswordControllerTests
+    {
+        private ForgotPasswordController controller;
+        private IPasswordResetService passwordResetService;
+
+        [SetUp]
+        public void SetUp()
+        {
+            passwordResetService = A.Fake<IPasswordResetService>();
+
+            controller = new ForgotPasswordController(passwordResetService);
+        }
+
+        [Test]
+        public void No_model_should_render_basic_form()
+        {
+            // When
+            var result = controller.Index(model: null);
```

Comment:
Could this just be Index()? Also, I'm trying to think of a more intuitive test name, but my mind is blank haha! Maybe just Renders_basic_form, as "No model" can be a bit confusing if you just scan over the test names?

Reply:
It looks like the Index(model: null) is necessary to specify between the HttpGet Index() action and the HttpPost Index() action as they both take a single nullable input, otherwise NUnit doesn't know which one to test when nothing is passed in. I have renamed it "Render_basic_form_when_there_is_no_error_model" which should help with the name and explain the null model, if that is satisfactory?

Reply:
Sounds good to me! :)